images/linux/scripts/installers/git.sh

```diff
@@ -15,6 +15,11 @@ add-apt-repository $GIT_REPO -y
 apt-get update
 apt-get install git -y
 git --version
+# Git version 2.35.2 introduces security fix that breaks action\checkout https://github.com/actions/checkout/issues/760
+cat <<EOF >> /etc/skel/.gitconfig
+[safe]
+        directory = *
+EOF
```

Comment:
Would it not make more sense to put this into `/etc/gitconfig`? Or would that break usage in containers?

Reply:
@thboop @TingluoHuang do you have any thoughts regarding containers?

Reply:
`/etc/gitconfig` should be fine since we just want to turn off the `safe.dir` enforcement in the hosted image.